workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstancePage.tsx

```diff
@@ -201,8 +203,10 @@ export const WorkflowInstancePage = ({
   return (
     <BaseOrchestratorPage
       title={value?.instance.processId ?? value?.instance.id ?? instanceId}
-      type="Workflow runs"
-      typeLink="/orchestrator/instances"
+      type={
+        fromPage === '/orchestrator/instances' ? 'All runs' : 'Workflow Runs'
```

Comment:
When touching this part, can we use `workflowInstancesRouteRef` instead of the hardcoded string?

Reply:
The router is relative to /orchestrator. how's this?
![image](https://github.com/user-attachments/assets/6a4b28f5-719d-4a7e-bf3c-9f873db2c591)

Reply:
Did we agree on this text change with UX?

Reply:
I see we did